tests/utils/test_client.py

```diff
@@ -44,6 +44,21 @@ def test_store_auth_cookie():
     assert my_client.session.auth is None
 
 
+@mock_api()
+def test_passing_base_path():
+    base_path = "/api/testing/base/path/"
+    my_client = utils.client.create(**CONFIG["my_instance_name"], base_path=base_path)
+    assert my_client.base_path == base_path
+
+
+@mock_api()
+def test_passing_session():
+    my_client = utils.client.create(**CONFIG["my_instance_name"])
+    session = my_client.session
+    my_other_client = utils.client.create(**CONFIG["my_portless_instance"], session=session)
+    assert my_other_client.session == session
+
+
```

Comment:
These tests don't need `@mock_api` since they aren't actually making any calls to Tamr.  Can you remove these decorators and delete the empty response log files that were generated?

Reply:
Will do in next commit. Thanks for the review Sam!